styleguide/source/assets/scss/01-atoms/_image.scss

```
@@ -18,8 +18,10 @@
       @include gutter($margin-right-half...);
     }
 
-    &--right {
+    &--right {;
```

@designerneil remove semicolon.
```suggestion
    &--right {
```

---

good catch!